File: 2021/JAN21C/ANTSCHEF.py
for _ in range(int(input())):
    n = int(input())
    lines = []
    line = list(map(int, input().split())
    positives = 0
    negatives = 0
    for i in range(1, len(line)):
        number = line[i]
        if number > 0:
            positives += 1
        else:
            negatives += 1

    print(positives*negatives)

File: 2021/JAN21C/ANTSCHEF.cpp
#include<iostream>
using namespace std;
int main(){
    int t;
    cin>>t;
    while (t--)
    {
        int n;
        cin>>n;
        int positives = 0, negatives = 0;
        while(n--){
            int m;
            cin>>m;
            while(m--){
                int x;
                cin>>x;
                if(x>0)
                    positives += 1;
                else
                {
                    negatives += 1;
                }
            }
        }
        cout<<positives*negatives<<'\n';
    }
    
}